app/_locales/en/messages.json

```diff
@@ -578,6 +578,12 @@
   "editContact": {
     "message": "Edit Contact"
   },
+  "editNonceField": {
+    "message": "Edit Nonce"
+  },
+  "editNonceMessage": {
+    "message": "Use this advanced feature to cancel or speed up older transactions.  "
```

Comment:
I don't think we should use this copy, as it encourages the exact usage that we want to discourage. I've just  added a comment in the Figma design about this - we can discuss it further there.

Reply:
There is new copy available in the Figma design now